app/javascript/includes/remote_form.js

```diff
@@ -114,13 +114,13 @@ function applyFormSemanticErrors(form, errors) {
     const errorLines = []
 
     Object.keys(errors).forEach(function (errorName) {
-        const errorMessage = errors[errorName].join(', ')
+        const errorMessage = errorName == 'base' ? errors[errorName] : errors[errorName].join(', ')
```

Comment:
it looks wrong

errors that being send by server should always have following format

```
{
  attribute_or_base: [error_message, ...],
  ...
}
```

if `errors[errorName]` is not an array than it' a server-side bug